public/index.html

```diff
@@ -998,6 +998,13 @@
 					</div>
 					<div class="grid-item col-xl-12 mb-3 text-center" ng-cloak>
 						<button class="btn btn-primary btn-sm btn-download" ng-click="downloadZip()"><img src="assets/img/download.svg" alt="Download"> Generated config <small>(.zip)</small></button>
+						<button tooltips tooltip-template="Copied!" tooltip-show-trigger="click" class="btn btn-success btn-sm btn-download btn-base64" ng-click="downloadBase64()"><img src="assets/img/download.svg" alt="Download"> Copy Base64 <small>(inline)</small></button>
+					</div>
+					<div id="base64-zip-wrapper" class="custom-control-input">
+						<button class="btn btn-light btn-clipboard" id="btn-base64-zip-line" ngclipboard data-clipboard-target="#base64-zip-line" ngclipboard-success="clipboardSuccess('base64-zip-line')">
+							<img src="assets/img/clipboard-dark.svg" alt="Copy to clipboard">
+						</button>
+						<div id="base64-zip-line"></div>
```

Comment:
There are missing styles to make this block with the text invisible

Reply:
in parent item i have class custom-control-input, this is opacity, position and z-index styles
```suggestion
					<div id="base64-zip-wrapper" class="custom-control-input">
```

Reply:
in parent item i have class custom-control-input, this is opacity, position and z-index styles